README.md

```diff
@@ -0,0 +1,176 @@
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[responsiveness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- An IDE (Intergrated Development Environment ) such as vsode or atom
+- A version control system such as git
+- Node
+
+### Setup
+
+1.  Clone the Repo with:
+
+```
+git clone https://github.com/maths29/Capstone-project-1.git
+```
+
+2. switch to the cloned directory
+
+```
+cd capstone project-1
+```
+
+### Installation
+
+1. Install the project with it's depenedcies
+
+```
+npm install
+```
+
+Then open the live server or open the `index.html` file with your browser
+
+### Usage
+
+- this is the portfolio project I'll use to update more changes.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "**/*.{css,scss}"
+- npx eslint . 
+
+<!-- Deployment -->
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+- <a href=" https://maths29.github.io/Capstone-project-1/">[🚀 Live Demo Link ] </a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Iwu John Chinonso**
+
+- GitHub: [@githubhandle](https://github.com/maths29)
+- Twitter: [@twitterhandle](https://twitter.com/maths29)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/iwu-john-b92b01148/ )
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added:
+
+- [ ] **[Project: User Authentication]**
+- [ ] **[Project: Online Voting]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the <a href="https://github.com/maths29/Capstone-project-1./issues"> issues page </a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd like to thank and give credit to  [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). for an inspiring Original desing and idea.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./mit.md) licensed.
```

Comment:
Great job @maths29 🤝

- [ ] Kindly fix the broken MIT license link to make the readme file more professional.